spec/event_mapping_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+RSpec.describe "event_mapping" do
+  before :all do
+    @plan = plan(configuration_directory: 'examples')
+  end
+
+  it "renders a plan without errors" do
+    expect(@plan).to be_a(RubyTerraform::Models::Plan)
+  end
+
+  context "resource creation" do
+    let(:event_rules) { @plan.resource_changes_matching(type: "aws_cloudwatch_event_rule") }
+
+    it "creates aws_cloudwatch_event_rules for bus targets" do
+      expect(@plan).to include_resource_creation(type: 'aws_cloudwatch_event_rule').twice
+    end
+
+    it "points to specified bus" do
+      expect(@plan).to include_resource_creation(type: 'aws_cloudwatch_event_rule')
+        .with_attribute_value(:event_pattern, "{\"detail-type\":[\"event.DementorsAppear\"]}")
```

Comment:
this is nice that it validates the actual output json (regarding my comment about the `<>` being escaped in highwingio/enrichment-component#15. Lets see how it runs but it might be nice to add some helpers to evaluate this more natively in ruby.

```suggestion
        .with_attribute_value(:event_pattern, {"detail-type" => ["event.DementorAppear"]}.to_json)
```